src/components/include/media_manager/media_manager.h

```diff
@@ -40,9 +40,6 @@ namespace media_manager {
 
 class MediaManager {
  public:
-  virtual void PlayA2DPSource(int32_t application_key) = 0;
-  virtual void StopA2DPSource(int32_t application_key) = 0;
-
```

Comment:
Deprecate instead of remove, we need to wait for a major release to remove public methods

Reply:
@jacobkeeler the following changes were done:
1. Merge latest develop branch in https://github.com/smartdevicelink/sdl_core/pull/2774/commits/40c3b879215024e38ef232836a6c95a26beeece6
2. Reverted changes related to this PR in https://github.com/smartdevicelink/sdl_core/pull/2774/commits/7cf8f53359cba5a9e9b4ad0dcb1cba40fa1bce1a
3. Mark redundant code as deprecated in https://github.com/smartdevicelink/sdl_core/pull/2774/commits/07d5fe0f02a84dc6488dc569ce00835fb0f50c9a

Not sure if I need to remove related unit tests for now or should they be removed together with deprecated functions

Reply:
@jacobkeeler removed in https://github.com/smartdevicelink/sdl_core/pull/2774/commits/c1d69b64dc6cbf84d94c0a35444273599cb9666a